src/status_im/ui/screens/chat/components/input.cljs

```diff
@@ -587,36 +587,38 @@
                                       (reanimated/set-shared-value bg-opacity (reanimated/with-timing 0)))
                                     (reanimated/set-shared-value translate-y (reanimated/with-timing (- y)))
                                     (reanimated/set-shared-value shared-height (reanimated/with-timing (min y max-height)))))
-              [reanimated/view {:style (reanimated/apply-animations-to-style
-                                        {:height shared-height}
-                                        {})}
+              [:<>
+               [reply/reply-message-auto-focus-wrapper text-input-ref]
+               [reanimated/view {:style (reanimated/apply-animations-to-style
+                                         {:height shared-height}
+                                         {})}
                ;;INPUT MESSAGE bottom sheet
-               [gesture/gesture-detector {:gesture bottom-sheet-gesture}
-                [reanimated/view {:style (reanimated/apply-animations-to-style
-                                          {:transform [{:translateY translate-y}]}
-                                          (styles/new-input-bottom-sheet window-height))}
+                [gesture/gesture-detector {:gesture bottom-sheet-gesture}
+                 [reanimated/view {:style (reanimated/apply-animations-to-style
+                                           {:transform [{:translateY translate-y}]}
+                                           (styles/new-input-bottom-sheet window-height))}
                  ;handle
-                 [rn/view {:style (styles/new-bottom-sheet-handle)}]
-                 [rn/view {:style {:height (- max-y 80)}}
-                  [text-input {:chat-id                chat-id
-                               :on-content-size-change input-content-change
-                               :sending-image          false
-                               :refs                   refs
-                               :set-active-panel       #()}]]]]
+                  [rn/view {:style (styles/new-bottom-sheet-handle)}]
+                  [rn/view {:style {:height (- max-y 80)}}
```

Comment:
so it should be here, between the handle and input text, and we need to subtract from input height, and also we need to account for changes in translate-y , so we need to move drawer when we add new component in it

Reply:
Thanks @flexsurfer. Fixed it.